actions/actions.py

```diff
@@ -123,6 +125,13 @@ def slot_mappings(self) -> Dict[Text, Union[Dict, List[Dict]]]:
                 self.from_entity(intent="inform", entity="province"),
                 self.from_text(),
             ],
+            "location": [self.from_entity(entity="text"), self.from_text()],
```

Comment:
I don't think that we have a `text` entity, so this is should probably just be `[self.from_text()]`